### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by marurevi**
#### Add style to Bookstore app

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -12,28 +13,48 @@ nav {
 
 nav > * {
```
Comment (hiromataba) [inline src/App.css]: - [x] You have a great navbar but it seems that you forgot the `user` icon on the top right corner also your books and category links do not have the text-decoration please check the original design

Your design 👇 
Screen Shot 2022-06-10 at 20 49 54

The original Design 👇 

Screen Shot 2022-06-10 at 21 05 31

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @marurevi     :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (marurevi) [reply src/App.css]: @hiromataba I added the user but it seems you already approve it. You can take a look if you like. Thanks! 😉

Comment (hiromataba) [reply src/App.css]: @marurevi i can see it yeah, Great design 😃

**Pull request merged**